addons/website_slides/models/slide_slide.py

```diff
@@ -229,7 +229,7 @@ class Slide(models.Model):
     nbr_infographic = fields.Integer("Number of Images", compute='_compute_slides_statistics', store=True)
     nbr_article = fields.Integer("Number of Articles", compute='_compute_slides_statistics', store=True)
     nbr_quiz = fields.Integer("Number of Quizs", compute="_compute_slides_statistics", store=True)
-    total_slides = fields.Integer(compute='_compute_slides_statistics', store=True)
+    total_slides = fields.Integer(compute='_compute_slides_statistics', store=False, compute_sudo=True)
```

Comment:
Any reason to "unstore" this? It was probably stored for a reason (sorting, search, ...).

Reply:
So the change we made with completion time and total slides is the following: When you are an administrator, you see both published and unpublished slides, so total_slides should show you the value that is sum of all of these slides. For portal user, total_slides should calculate only published slides. The easiest way to do it that me and Florian could think of was to recalculate total_slides value every time user opens a page. This way value will be calculated every time page opens based on the level of access rights of a user, and at the same time whenever admin publishes or unpublished courses, portal user will get updated in comparatively "real time"